pkg/oc/bootstrap/docker/up.go

```diff
@@ -1039,6 +1041,34 @@ func (c *ClientStartConfig) InstallServiceCatalog(out io.Writer) error {
 	return c.OpenShiftHelper().InstallServiceCatalog(f, c.LocalConfigDir, publicMaster, openshift.CatalogHost(c.RoutingSuffix, c.ServerIP), c.imageFormat())
 }
 
+func (c *ClientStartConfig) ShouldInstallTemplateServiceBroker() bool {
+	if c.ImageVersion == "latest" {
+		return true
+	}
+
+	// the TSB, which requires 3.7
+	serverVersion, _ := c.OpenShiftHelper().ServerPrereleaseVersion()
+	if serverVersion.LT(openshiftVersion37) {
+		return false
+	}
+	return true
+}
+
+// InstallTemplateServiceBroker will start the installation of template service broker
+func (c *ClientStartConfig) InstallTemplateServiceBroker(out io.Writer) error {
+	f, err := c.Factory()
+	if err != nil {
+		return err
+	}
+	publicMaster := c.PublicHostname
+	if len(publicMaster) == 0 {
+		publicMaster = c.ServerIP
+	}
+	// TODO we want to use this eventually, but until we have our own image for TSB, we have to hardcode this origin
+	//return c.OpenShiftHelper().InstallTemplateServiceBroker(f, c.imageFormat())
+	return c.OpenShiftHelper().InstallTemplateServiceBroker(f, c.Image)
```

Comment:
@jim-minter I had to hardcode the image format to be our base image.  See the comment above.